homeassistant/components/alexa/entities.py

```diff
@@ -291,14 +292,14 @@ def alexa_id(self):
         """Return the Alexa API entity id."""
         return generate_alexa_id(self.entity.entity_id)
 
-    def display_categories(self):
+    def display_categories(self) -> list[str] | None:
         """Return a list of display categories."""
         entity_conf = self.config.entity_config.get(self.entity.entity_id, {})
         if CONF_DISPLAY_CATEGORIES in entity_conf:
             return [entity_conf[CONF_DISPLAY_CATEGORIES]]
         return self.default_display_categories()
 
-    def default_display_categories(self):
+    def default_display_categories(self) -> list[str] | None:
```

Comment:
I have made this optional due to the missing return statement in `BinarySensorCapabilities`, but maybe it should be compulsory.

Reply:
same here, it can be compulsory I think as it sees that `get_type()` for binary sensor is returning `TYPE_CONTACT` by default.

Reply:
It seems the return value can be `None`, the Alexa capabilities are only available for certain device classes.